### PR TITLE
Added FMW Integration test with Dynamic Cluster using WLST

### PR DESCRIPTION
**Pull request by hzhao-github**
Create a FMW domain with Dynamic Cluster using WLST

Jenkins jobs:

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/4223 (parallel)
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/4224 (sequential)
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/4449  (sequential)

### CODE REVIEW

**Review by anpanigr — COMMENTED** [review]
Replace the occurrence of JRF with FMW to reflect the image name

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDynamicDomainInPV.java [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+  }
+
+  /**
+   * Create a basic JRF dynamic cluster model in image domain.
+   * Verify Pod is ready and service exists for both admin server and managed servers.
+   * Verify EM console is accessible.
+   */
+  @Test
+  @DisplayName("Create JRF Dynamic Domain in PV")
+  public void testJrfDynamicDomainInPV() {
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDynamicDomainInPV.java:152]: testJrfDynamicDomainInPV()  --> testFmwDynamicDomainInPV()

Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDynamicDomainInPV.java]: changed

**Review by rjeberhard — APPROVED** [review]

**Comment by rjeberhard** [comment]
Since @anpanigr is out, I'm going to assume his approval because he provided some comments and they were all addressed.

**Pull request merged**